sources/packages/web/src/components/common/modals/ConfirmExtendTime.vue

```diff
@@ -0,0 +1,116 @@
+<template>
```

Comment:
The "Yes" button here seems to me as the wrong or dangerous choice for the user. IMO it would be the preferred.

![image](https://user-images.githubusercontent.com/61259237/139496679-4c6fedf1-7cdf-44e9-8b01-b7dfb6701521.png)

Reply:
nice catch